connector_ambumovil/models/service.py

```diff
@@ -48,21 +48,26 @@ def get_ambulance_stock(self, code):
         return product_stock_ld
 
     @api.model
-    def consume_ambulance_stock(self, code, service_num, moves, employees, validate=True):
+    def consume_ambulance_stock(self, code, service_num, moves, employees, out=True, validate=True):
```

Comment:
I think it twice and I think it'd be clearer if we call the parameter "refund", by default refund=False. Could you change it please? Thanks!! :-)